maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -764,6 +766,40 @@
     @Parameter( property = "dependenciesToScan" )
     private String[] dependenciesToScan;
 
+    /**
+     * <p>
```

Comment:
Here we should say "Since 3.0.0-M5 and Maven 3.3.x".

Reply:
We should write the documentation in the file `java9.md`, otherwise the people would not know about our work here. So we should add a new entry especially the toolchain in `maven-surefire-plugin/src/site/sitre.xml` and the child in the folder `maven-surefire-plugin/src/site/apt/examples`.

Reply:
The part I don't get is what this has to do with java 9 ... toolchains are not limited to java 9, so why would someone be reading that page to figure out how to use toolchains with surefire?

I can understand making a new page in examples and adding a link to the site, and possibly to the java 9 page as well.

Reply:
feel free to a new page.
this would be very valuable work

Reply:
done